packages/terra-clinical-item-view/src/terra-dev-site/doc/clinical-item-view/AccessibilityGuide.4.doc.mdx

```diff
@@ -0,0 +1,230 @@
+
+#### Styling Override
+
+Clincial Item View takes in Clinical Item Displays and by default the Item View has its own styling for these display elements.
+However, the Item Displays have several different types of styling that can be applied to each individual display. Sometimes this styling conveys important information, like a display with a strikethrough.
+
+To allow the usage of custom styling we have the `overrideDefaultStyling` prop.
+
+This prop allows the user to override the Item View default styling and instead use the styling directly applied to the Item Displays.
+If the override prop is set to true but the displays have no specific styling set, the displays will end up with the default Item Display styling.
+More information on these styling types can be found on the Clincial Item Display [**page**](https://engineering.cerner.com/terra-clinical/components/terra-clinical-item-display/clinical-item-display/clinical-item-display).
+
+#### Truncation Handling
+
+There is an `isTruncated` prop for Clinical Item View, when this is set to true all displays and comments in the Item View can be truncated. Otherwise, the content wraps to a newline when truncation is false.
+The actual Item Displays themselves also have an `isTruncated` prop. If the Item View has truncation set to false but some of the Item Displays have it set to true, only those specific displays will be truncated. The rest will have the word wrap behavior.
+
+<Notice variant="important" ariaLevel="3">
+
+##### Truncated Text
+
+Truncation of text can pose an accessibility concern if no method of disclosing the full text is availableto the user.
+When using `isTruncated`, consumers are responsible for providing a progressive disclosure pattern to disclose the full Item Display text in order to ensure that it is accessible for keyboard navigation users.
+
+**There should always be a method of accessing the truncated information. If there is no way to progressively disclose the full content of the truncated information, then truncation should not be used.**
+
+Some examples of progressive disclosure patterns that may be used to disclose truncated information include:
+- Accordions
+- Dialogs
+- Popovers
+- Show/Hide
+- Split Views
+- Toasts
+
+The method of disclosure **must** be accessible via keyboard interactions.
+
+Truncation should be avoided where it is not necessary. Certain content should **never** be truncated (i.e. medication names and dosages in menus where the user is selecting from a list of choices).
+
+</Notice>
+
+### Other Considerations
+
+#### Clinical Item View Unordered List Structure
+
+Displays in the Item View are rendered as list items inside a primary unordered list.
+The structure is different for the different types of layouts - one column, two columns, and two columns by row. Their structure is as follows:
+
+**For a one column layout with 8 displays**
```

Comment:
I wonder if we can display these 3 lists as side by side for better comparison.

Reply:
I played around with it a bit, does this seem better?

<img width="801" alt="Screenshot 2023-09-22 at 1 35 47 PM" src="https://github.com/cerner/terra-clinical/assets/40574651/4e7bc704-71bf-4817-84c1-47f503081ca9">

Reply:
I can also top align them
<img width="801" alt="Screenshot 2023-09-22 at 1 39 26 PM" src="https://github.com/cerner/terra-clinical/assets/40574651/8bdebb88-c015-4540-9145-28e926d96227">

Reply:
That looks good!

Reply:
Sweet, added here: https://github.com/cerner/terra-clinical/pull/916/commits/5e73fd5df78fae109565a8691764b0764055e7c1